### ISSUE TITLE
Test suite doesn't compile with crypto-api 0.11

### ISSUE THREAD

**Issue by snoyberg** [prowdsponsor/crypto-conduit#16]
I haven't had a chance to look into this in depth yet, but I got the error report during a Stackage build. Here's the message:

```
Unpacking to crypto-conduit-0.5.0/
Resolving dependencies...
Configuring crypto-conduit-0.5.0...
Building crypto-conduit-0.5.0...
Preprocessing library crypto-conduit-0.5.0...
[1 of 1] Compiling Crypto.Conduit   ( src/Crypto/Conduit.hs, dist/build/Crypto/Conduit.o )
In-place registering crypto-conduit-0.5.0...
Preprocessing test suite 'runtests' for crypto-conduit-0.5.0...
[1 of 1] Compiling Main             ( tests/runtests.hs, dist/build/runtests/runtests-tmp/Main.o )

tests/runtests.hs:120:8:
    Ambiguous occurrence `C.ecb'
    It could refer to either `C.ecb',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.ecb',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:125:8:
    Ambiguous occurrence `C.unEcb'
    It could refer to either `C.unEcb',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.unEcb',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:131:14:
    Ambiguous occurrence `C.cbc'
    It could refer to either `C.cbc',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.cbc',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:136:14:
    Ambiguous occurrence `C.unCbc'
    It could refer to either `C.unCbc',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.unCbc',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:142:14:
    Ambiguous occurrence `C.cfb'
    It could refer to either `C.cfb',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.cfb',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:147:14:
    Ambiguous occurrence `C.unCfb'
    It could refer to either `C.unCfb',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.unCfb',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:153:14:
    Ambiguous occurrence `C.ofb'
    It could refer to either `C.ofb',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.ofb',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:158:14:
    Ambiguous occurrence `C.unOfb'
    It could refer to either `C.unOfb',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.unOfb',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:164:14:
    Ambiguous occurrence `C.ctr'
    It could refer to either `C.ctr',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.ctr',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:169:14:
    Ambiguous occurrence `C.unCtr'
    It could refer to either `C.unCtr',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.unCtr',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:175:20:
    Ambiguous occurrence `C.ctr'
    It could refer to either `C.ctr',
                             imported qualified from `Crypto.Classes' at tests/runtests.hs:14:1-36
                          or `C.ctr',
                             imported qualified from `Crypto.Modes' at tests/runtests.hs:16:1-34

tests/runtests.hs:181:48: Not in scope: `C.cbcMac'
```

**Comment by meteficha** [comment]
Does this still happen?

**Comment by snoyberg** [comment]
Yes, last build was yesterday and I have the same test error log.

**Comment by meteficha** [comment]
Should be fixed on `0.5.1`.
